styledComponents.js

```diff
@@ -1,4 +1,17 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
+
+export const StyledSignedOutContainer = styled.div`
+  opacity: 0.4;
+  pointer-events: none;
```

Comment:
you use this styling conditionally with a `$disabled` prop and use the `isLoggedOut` boolean to toggle this behavior.